app/lib/check_answers_fields.yml

```diff
@@ -56,6 +56,7 @@ sections:
           type: boolean
         - attribute: employment_status
           type: select
+          skip_unless: self_employed_flag_disabled?
```

Comment:
There's sort of a double negative here (`unless` + `disabled`), which makes this a bit hard to read. You _could_ change this to `skip_if: self_employed_flag_enabled?`, if you were prepared to update `CheckAnswers::SectionListerService#build_field` with a line like:

 ```
return if field_data[:skip_if].present? && @model.send(field_data[:skip_if])
```

Very possibly overkill though!